docs/sources/configuration/_index.md

```diff
@@ -1900,10 +1900,15 @@ fifocache:
   # CLI flag: -<prefix>.fifocache.max-size-items
   [max_size_items: <int> | default = 0]
 
-  # The expiry duration for the cache.
+  # The expiry duration for the cache; deprecated in favour of `-<prefix>.fifocache.ttl`.
```

Comment:
This suggestion is made to attempt to match another deprecation notice on a configuration knob.

```suggestion
  # Deprecated: The expiry duration for the cache. Use `-<prefix>.fifocache.ttl`.
```